testsuite/classlibrary/TestAbstractFunction.sc

```diff
@@ -0,0 +1,137 @@
+TestAbstractFunction : UnitTest {
+
+	test_rest_binop {
+		var func;
+		var args = [[1, Rest(1)], [Rest(1), 1], [Rest(1), Rest(1)], [1, Rest()], [Rest(), Rest()]];
+		var funcs = [
+			{ |a, b| a * b },
+			{ |a, b| a + b },
+			{ |a, b| a - b },
+			{ |a, b| a / b }
+		];
+		var all = args.collect { |pair|
+			funcs.collect { |f| f.valueArray(pair) }
+		};
+		all.postln;
+
+		this.assert(
+			all.flat.every(_.isKindOf(Rest)),
+			"All binary math operations on Rest should return Rest.\n"
```

Comment:
This test could be more helpful. If one of these operations fail, we won't know which one it was.